xaod_cpp_transformer/transformer.py

```diff
@@ -27,22 +27,22 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import json
 import logging
+import logstash
 import os
 import time
-import timeit
 from collections import namedtuple
 import re
 
 import psutil
-import uproot
+# import uproot
 
 from servicex.transformer.servicex_adapter import ServiceXAdapter
 from servicex.transformer.transformer_argument_parser import TransformerArgumentParser
 from servicex.transformer.object_store_manager import ObjectStoreManager
 from servicex.transformer.rabbit_mq_manager import RabbitMQManager
-from servicex.transformer.uproot_events import UprootEvents
-from servicex.transformer.uproot_transformer import UprootTransformer
-from servicex.transformer.arrow_writer import ArrowWriter
+# from servicex.transformer.uproot_events import UprootEvents
```

Comment:
Please don't comment out code. We have the git commit history if for some reason we need this back

Reply:
agree